packages/components/src/local/mapping/index3.stories.tsx

```diff
@@ -16,7 +16,7 @@ import { HexGrid } from '../hex-grid'
 // import data types
 import { Phase, serverPath } from '@serge/config'
 
-import data from './data/atlantic-cells';
+import data from './data/atlantic-cells'
```

Comment:
@sanjayradadiya - if you put a package.json in `data`, you can call it `@serge/data`, then this import becomes `@serge/atlantic-cells` (I think...).  That's the way we do it in the rest of the app.

Reply:
ohhh, yes i forget that

Reply:
this are different one it is the sample data inside mapping component `atlantic-cells.ts` file